scripts/close_inactive.rb

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env ruby
+require 'octokit'
+
+REPOS = ["theforeman/foreman"]
+KEEP = 6 # months
```

Comment:
> Consistency within the project.

Done, in addition to that, you can define
`'config/local_close_inactive.yaml'` (an gitignored file) to override
defaults.

Also it's only touch WoC PRs now. I need to add filtering to prevent
iterating them all now and I guess it's all.

-- 
Later,
 Lukas #lzap Zapletal